hubconf.py

```diff
@@ -0,0 +1,21 @@
+# autopep8: off
+dependencies = ["torch", "torchvision"]
+
+import sys
+from pathlib import Path
+repo_dir = Path(__file__).parent
+multimodal_src_dir = repo_dir / "hi-ml-multimodal" / "src" / "health_multimodal" / "image"
+sys.path.append(str(multimodal_src_dir))
+
+from model import ImageModel
+from model import get_biovil_resnet as _biovil_resnet
+# autopep8: on
+
+
+def biovil_resnet(pretrained: bool = False) -> ImageModel:
```

Comment:
Could we please add a unit-test comparing the models initialised with (I) our old resnet code and (II) torch-hub code, running them on some dummy torch rand (3, 448, 448) array and comparing their outputs?

Reply:
Not only the shapes, but also the data content.

Reply:
If we add this test now, it will need to reference the branch of this PR (https://github.com/microsoft/hi-ml/pull/543#issue-1320507174) and will fail once it's merged. I suggest the following instead:

1. Merge this PR
2. Open an issue about that test
3. Open a PR with the new test
4. Merge the new PR and close the issue

What do you think?

Reply:
If this is the new and suggested way of loading biovil resnet, then we should it test and make sure that the forward pass outputs are matched. The worst-case scenario would be the following: The PR is merged, someone pulls the models from torch-hub or checks out to this commit and runs experiments with the wrong model. I think we probably don't want this corner case.

Something like the following, it may be syntactically incorrect but gives the basic idea.
```
import torch
github = 'microsoft/hi-ml:add-hubconf'
model1 = torch.hub.load(github, 'biovil_resnet', pretrained=True)
model2 = "<your_old_way_of_loading_it>
dummy_input = torch.rand(...)
assert torch.all(model1(dummy_input), model2(dummy_input), tol=...)
```

Reply:
Context for @Shruthi42:

I will add the suggested test pointing to this branch. If the CI passes, I'll remove the name of the branch, which will make the CI fail again, but will be ready to merge. I hope that makes sense.